internal/cli/clankcli/style.go

```
@@ -7,15 +7,15 @@ import "github.com/charmbracelet/lipgloss"
 //
-// Colours follow standard 16-colour ANSI semantics: 9=red, 10=green,
-// 11=yellow, 12=bright-blue. lipgloss strips them automatically when
-// stdout isn't a tty (via termenv), so piping to a file / another
-// process is safe.
+// Colours follow standard 16-colour ANSI semantics: 8=grey, 9=red,
+// 10=green, 11=yellow, 12=bright-blue. lipgloss strips them automatically
+// when stdout isn't a tty (via termenv), so piping remains safe.
 var (
 	styleOK          = lipgloss.NewStyle().Foreground(lipgloss.Color("10")).Bold(true) // green
 	styleWarn        = lipgloss.NewStyle().Foreground(lipgloss.Color("11"))            // yellow
 	styleErr         = lipgloss.NewStyle().Foreground(lipgloss.Color("9")).Bold(true)  // red
```

golangci-lint (2.12.2)

[error] var styleErr is unused

(unused)